src/utils/PersonIdentityUtils.ts

```diff
@@ -4,9 +4,38 @@ import { Logger } from "@aws-lambda-powertools/logger";
 import { MessageCodes } from "../models/enums/MessageCodes";
 import { AppError } from "./AppError";
 import { HttpCodesEnum } from "./HttpCodesEnum";
+import { Name } from "./IVeriCredential";
 
 export const personIdentityUtils = {
 
+	getNamesFromYoti(givenName: string, familyName: string): Name[] {
+		const givenNames = givenName.split(/\s+/);
```

Comment:
would be good to add an exists checks for giveName before calling split for some defensive coding

Reply:
We do a check before the function is called, https://github.com/alphagov/di-ipv-cri-f2f-api/blob/F2F-1000/src/services/YotiCallbackProcessor.ts#L178 so I wonder if adding a second check in this function might be a bit overkill? It doubles the amount of code we have to maintain (eg if we need to change this check or something). Happy to have a discussion about it though